From bfb2121ee5e634260773bed0d9619895d6ca517e Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Mon, 18 Dec 2023 20:45:27 +0100
Subject: [PATCH 0359/1695] dts: stm32: fix RTC node at STM32MP1 SoC level

STM32MP15 RTC does not generate interrupt whereas STM32MP13 RTC
consume EXIT interrupt 19.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 090d1937c3a76014ff2e2781b9a771187cc4e68a dts: stm32: fix RTC node at STM32MP1 SoC level]
Change-Id: I823d9f25a5e2dba1f14a437f3456bac6ca0ee7e0
Signed-off-by: Pascal Paillet <p.paillet@st.com>
Signed-off-by: Lionel Debieve <lionel.debieve@foss.st.com>
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/349070
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Domain-Review: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/arch/arm/dts/stm32mp131.dtsi | 1 +
 core/arch/arm/dts/stm32mp151.dtsi | 1 -
 2 files changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/arch/arm/dts/stm32mp131.dtsi b/core/arch/arm/dts/stm32mp131.dtsi
index 776ba1bce..b7347f41b 100644
--- a/core/arch/arm/dts/stm32mp131.dtsi
+++ b/core/arch/arm/dts/stm32mp131.dtsi
@@ -271,6 +271,7 @@
 			reg = <0x5c004000 0x400>;
 			clocks = <&rcc RTCAPB>, <&rcc RTC>;
 			clock-names = "pclk", "rtc_ck";
+			interrupts-extended = <&exti 19 IRQ_TYPE_LEVEL_HIGH>;
 			status = "disabled";
 		};
 
diff --git a/core/arch/arm/dts/stm32mp151.dtsi b/core/arch/arm/dts/stm32mp151.dtsi
index 13efb4143..4c0b3bcba 100644
--- a/core/arch/arm/dts/stm32mp151.dtsi
+++ b/core/arch/arm/dts/stm32mp151.dtsi
@@ -1523,7 +1523,6 @@
 			reg = <0x5c004000 0x400>;
 			clocks = <&rcc RTCAPB>, <&rcc RTC>;
 			clock-names = "pclk", "rtc_ck";
-			interrupts-extended = <&exti 19 IRQ_TYPE_LEVEL_HIGH>;
 			status = "disabled";
 		};
 
-- 
2.47.2

